Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "appflowy-cloud"
 version = "0.1.0"
-edition = "2021"
+edition = "2024"
```

Comment:
Do we need to upgrade the edition for rust-lib as well?

Reply:
Already done